### PR TITLE
fix: remove non-master "surfing" boards

### PR DESCRIPTION
**Pull request by TheYoctoJester**
The master branch is meant to "surf" on the current master of the Yocto Project respectively OpenEmbedded upstream, the majority of board integration are not following. In order to remove unneccessary clutter and save build time, just keep the core platforms
- QEMU
- Raspberry Pi
- BeagleBoard and their variants for the continuous work.

Changelog: Title
Ticket: None

### CODE REVIEW

File: meta-mender-qemu/README.md [hunk 1]
```diff
@@ -1,56 +0,0 @@
-# meta-mender-qemu
```
Comment (kacf) [inline meta-mender-qemu/README.md:1]: You said you were keeping qemu, did you mean to remove this?

File: meta-mender-raspberrypi/README.md [hunk 2]
```diff
@@ -1,68 +0,0 @@
-# meta-mender-raspberrypi
```
Comment (kacf) [inline meta-mender-raspberrypi/README.md:1]: You said you were keeping raspberrypi, did you mean to remove this?

File: scripts/mender-layers.xml [hunk 3]
```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (kacf) [inline scripts/mender-layers.xml:1]: It's not clear to me from the commit text why any of the files in `scripts` should be removed.

File: templates/local.conf.append [hunk 4]
```diff
@@ -1,71 +0,0 @@
-
```
Comment (kacf) [inline templates/local.conf.append:1]: It's not clear to me from the commit text why any of the files in `templates` should be removed.

**Comment by TheYoctoJester** [comment]
@kacf a catch-all explanation is due:
The `kas` files do not need the repo-based xml scripts nor templates. As the QEMU and Raspberry Pi "layers" do not hold any actual code, just repo xmls and templates again, they are not required anymore.

**Review by kacf — APPROVED** [review]
Thanks, that explains it!

**Pull request merged**